includes/class-wc-calypso-bridge-setup.php

```diff
@@ -166,6 +166,25 @@ public function is_theme_installed( $theme ) {
 	public function register_payments_welcome_page() {
 		global $menu;
 
+		// WooCommerce must be active.
+		if ( !is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
+			return;
+		}
+
+		// WC Payment must not be installed.
+		if ( array_key_exists( 'woocommerce-payments/woocommerce-payments.php', get_plugins() ) ) {
+			return;
+		}
+
+		// User country must be US.
+		if ( 'US' !== WC()->countries->get_base_country() ) {
+			return;
+		}
+
+		if ( 'yes' === get_option( 'wc_calypso_brdige_wcpay_welcome_page_opt_out', 'no' ) ) {
```

Comment:
The option name is a bit lengthy. Can someone suggest a better name :) ?

Reply:
Maybe something like this?

```suggestion
		if ( 'yes' === get_option( 'wc_calypso_bridge_payments_dismissed', 'no' ) ) {
```

Reply:
Yup! 👍  `dismiss` describes the user action better. I'll make an update.

Reply:
Updated in https://github.com/Automattic/wc-calypso-bridge/pull/703/commits/4059fe445ca392e315163040c95a8ee6e19c1f98